docs/configuration/http_conn_man/http_conn_man.rst

```diff
@@ -86,6 +87,8 @@ add_user_agent
 .. _config_http_conn_man_http_codec_options:
 
 http_codec_options
+  **DEPRECATED**, use :ref:`http2_settings <config_http_conn_man_http2_settings>` instead.
```

Comment:
Can you also update https://github.com/lyft/envoy/blob/master/DEPRECATED.md? Thanks.